transformation/plugins/org.jboss.tools.fuse.transformation.editor/src/main/java/org/jboss/tools/fuse/transformation/editor/wizards/NewTransformationWizard.java

```diff
@@ -434,14 +432,16 @@ private String generateModel(String filePath,
             return getJavaQualifiedName(filePath, monitor);
         }
         case JSON: {
+            String className = classNameBuilder.toString().replace("_", "");
```

Comment:
I fear to understand that with the current state, in the wizard there will be no error, some code will be generated but this code will not be used by the tooling after. So user, will be blocked in a further step? If it is the case, it seems even worse from a functional point of view. I would prefer to have a popup explaining to user that he can't use what he selected because there are underscores in it.